main.js

```diff
@@ -23,7 +23,15 @@ function isPalindrome(str) {
 /* 3. Return an integer in reverse
  * ex. reverseInt(521) === 125
  */
-function reverseInt(num) {}
+function reverseInt(num) {
```

Comment:
check if it is possible to use the `reverseString` method from question 1 for this

Reply:
Done